benchmark/launch_experiment.py

```diff
@@ -189,11 +200,48 @@ def main():
         )
 
     else:
-        env = MORecordEpisodeStatistics(mo_gym.make(args.env_id), gamma=args.gamma)
-        eval_env = mo_gym.make(args.env_id)
+        if "mario" in args.env_id:
+            env = mo_gym.make(args.env_id, death_as_penalty=True)
+            eval_env = mo_gym.make(args.env_id, death_as_penalty=True)
+        else:
+            env = mo_gym.make(args.env_id)
+            eval_env = mo_gym.make(args.env_id)
+        env = MORecordEpisodeStatistics(env, gamma=args.gamma)
+
         if "highway" in args.env_id:
             env = FlattenObservation(env)
             eval_env = FlattenObservation(eval_env)
+        elif "mario" in args.env_id:
+
+            def wrap_mario(env):
+                from gymnasium.wrappers import (
+                    FrameStack,
+                    GrayScaleObservation,
+                    ResizeObservation,
+                    TimeLimit,
+                )
+                from mo_gymnasium.envs.mario.joypad_space import JoypadSpace
+
+                from morl_baselines.common.utils import MaxAndSkipObservationV0
+
+                env = JoypadSpace(env, SIMPLE_MOVEMENT)
+                env = MaxAndSkipObservationV0(env, skip=4)
+                env = ResizeObservation(env, (84, 84))
+                env = GrayScaleObservation(env)
+                env = FrameStack(env, 4)
+                env = TimeLimit(env, max_episode_steps=5000)
+                return env
+
+            env = wrap_mario(env)
+            eval_env = wrap_mario(eval_env)
+
+            if args.record_video:
```

Comment:
Shouldn't be mario specific, I mean, we should be able to recordvideo of any env?